### PR TITLE
fix: describe bug

### PR DESCRIPTION
**Pull request by uriii3**
We detected a bug [CMT-126](https://cms-change.atlassian.net/browse/CMT-126) with the describe with the pre-release.

Although I solved it in my working branch, I think that maybe creating it with another branch and having it more clear in the release notes will be easier and clearer for the users and for us.

### CODE REVIEW

File: copernicusmarine/catalogue_parser/catalogue_parser.py [hunk 1]
```diff
@@ -196,12 +196,12 @@ def _construct_marine_data_store_product(
...
 
+    thumbnail_url = None
     if stac_product.assets:
         thumbnail = stac_product.assets.get("thumbnail")
         if thumbnail:
             thumbnail_url = thumbnail.get_absolute_href()
-        else:
-            thumbnail_url = None
+
     thumbnail = stac_product.assets and stac_product.assets.get("thumbnail")
```
Comment (renaudjester) [inline copernicusmarine/catalogue_parser/catalogue_parser.py:205]: This line is weird 🤔 Is it used? And it seems to do the same as the "if"

Comment (uriii3) [reply copernicusmarine/catalogue_parser/catalogue_parser.py:205]: Mmh no I think it is a blank line only, the remaining of the delete.

Comment (renaudjester) [reply copernicusmarine/catalogue_parser/catalogue_parser.py:205]: I am talking about: `thumbnail = stac_product.assets and stac_product.assets.get("thumbnail")`

Comment (uriii3) [reply copernicusmarine/catalogue_parser/catalogue_parser.py:205]: Ah, not sure about that 🤷

Comment (renaudjester) [reply copernicusmarine/catalogue_parser/catalogue_parser.py:205]: I think you can delete it!

Comment (uriii3) [reply copernicusmarine/catalogue_parser/catalogue_parser.py]: deleted!

**Review by renaudjester — APPROVED** [review]

**Pull request merged**